onnxruntime/featurizers_ops/cpu/forecasting_pivot_transformer.cc

```diff
@@ -0,0 +1,114 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "core/common/common.h"
+#include "core/framework/data_types.h"
+#include "core/framework/data_types_internal.h"
+#include "core/framework/op_kernel.h"
+
+#include "Featurizers/ForecastingPivotFeaturizer.h"
+#include "Featurizers/../Archive.h"
+
+namespace NS = Microsoft::Featurizer;
+namespace onnxruntime {
+namespace featurizers {
+
+template <typename T> //float, double
+struct ForecastingPivotTransformerImpl {
+  void operator()(OpKernelContext* ctx) const {
+    using TransformerT = Microsoft::Featurizer::Featurizers::ForecastingPivotTransformer<T>;
+    using MatrixT = NS::RowMajMatrix<T>;
+    using InputMatrixT = Eigen::Map<const MatrixT>;
+    using InputType = std::vector<InputMatrixT>;
+    using OutputType = std::vector<T>;
+
+    //Get the transformer
+    const auto* state_tensor(ctx->Input<Tensor>(0));
+    const uint8_t* const state_data(state_tensor->Data<uint8_t>());
+    Microsoft::Featurizer::Archive archive(state_data, state_tensor->Shape().GetDims()[0]);
```

Comment:
>.GetDims()[0] [](start = 76, length = 13)

nit: for the future .Size() is a better looking alternative.

Reply:
sure, will create another PR address that for all featurizers